charts/hivemq-platform-operator/templates/deployment.yml

```diff
@@ -24,10 +24,12 @@ spec:
     spec:
       {{- if .Values.podSecurityContext.enabled }}
       securityContext:
-        runAsUser: {{ .Values.podSecurityContext.runAsUser }}
-        runAsGroup: {{ .Values.podSecurityContext.runAsUser }}
-        fsGroup: {{ .Values.podSecurityContext.runAsUser }}
         runAsNonRoot: {{ .Values.podSecurityContext.runAsNonRoot }}
+        runAsUser: {{ .Values.podSecurityContext.runAsUser | default 185 }}
+        runAsGroup: {{ .Values.podSecurityContext.runAsGroup | default 0 }}
+        {{- if .Values.podSecurityContext.fsGroup }}
+        fsGroup: {{ .Values.podSecurityContext.fsGroup }}
+        {{- end }}
```

Comment:
we cannot set this `fsGroup` to be the same as the `runAsGroup`. On Openshift this value will be a range of GIDs, different than the `runAsGroup`.